buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle

```diff
@@ -28,11 +28,6 @@ dependencies {
     }
 }
 
-java {
-    sourceCompatibility = JavaVersion.VERSION_21
-    targetCompatibility = JavaVersion.VERSION_21
-}
```

Comment:
Although setting sourceCompatibility in the previous case didn't have any real effect, we decided to do it twice.